Sources/ClientRuntime/PrimitiveTypeExtensions/URLQueryItem+Extensions.swift

```diff
@@ -3,9 +3,9 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
-public typealias URLQueryItem = MyURLQueryItem
+public typealias URLQueryItem = SDKURLQueryItem
```

Comment:
The name `SDKURLQueryItem` is fine, but the objective is to not use the name `URLQueryItem` throughout our code because that name is easy to confuse with `Foundation.URLQueryItem` which has a similar name but different behavior.

So, this `typedef` has to go, and then that will require a slew more change, including in codegen.

Reply:
I'm going to approve and merge this.  I will eliminate the typedef & complete the renaming in a separate PR.